Tools/auterion/upload_skynode.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+DIR="$(dirname $(readlink -f $0))"
+DEFAULT_AUTOPILOT_HOST=10.41.1.1
+DEFAULT_AUTOPILOT_PORT=33333
+DEFAULT_AUTOPILOT_USER=auterion
+
+for i in "$@"
+do
+    case $i in
+        --file=*)
+        PX4_BINARY_FILE_ARGUMENT=" -f ${i#*=}"
+        ;;
+        --default-ip=*)
+        DEFAULT_AUTOPILOT_HOST="${i#*=}"
+        ;;
+        --default-port=*)
+        DEFAULT_AUTOPILOT_PORT="${i#*=}"
+        ;;
+        --default-user=*)
+        DEFAULT_AUTOPILOT_USER="${i#*=}"
+        ;;
+        --revert)
+        REVERT_AUTOPILOT_ARGUMENT=" -r"
+        ;;
+        --wifi)
+        DEFAULT_AUTOPILOT_HOST=10.41.0.1
+        ;;
+        *)
+            # unknown option
+        ;;
+    esac
+done
+
+# allow these to be overridden
+[ -z "$AUTOPILOT_HOST" ] && AUTOPILOT_HOST=$DEFAULT_AUTOPILOT_HOST
+[ -z "$AUTOPILOT_PORT" ] && AUTOPILOT_PORT=$DEFAULT_AUTOPILOT_PORT
+[ -z "$AUTOPILOT_USER" ] && AUTOPILOT_USER=$DEFAULT_AUTOPILOT_USER
+
+ARGUMENTS=()
+ARGUMENTS+=(-d $AUTOPILOT_HOST)
+ARGUMENTS+=(-p $AUTOPILOT_PORT)
+ARGUMENTS+=(-u $AUTOPILOT_USER)
+ARGUMENTS+=($PX4_BINARY_FILE_ARGUMENT)
+ARGUMENTS+=($REVERT_AUTOPILOT_ARGUMENT)
+
+echo "Flashing $AUTOPILOT_HOST ..."
+
+"$DIR"/remote_update_fmu.sh ${ARGUMENTS[@]}
```

Comment:
```suggestion
"$DIR"/remote_update_fmu.sh "${ARGUMENTS[@]}"
```

Reply:
I add this to b9c4dc8b97c81af9fdd2bea41554a33f3e3cd561.

Note that I left away all the quotes of the individual parts e.g. not doing `ARGUMENTS+=(${PX4_BINARY_FILE:+"-f $PX4_BINARY_FILE"})`. It works either with the individual parts or the entire argument list quoted but when doing both it just skips through flashing.
```
make px4_fmu-v5x upload_skynode_usb        
[0/1] Uploading PX4
Flashing 10.41.1.1 ...
```
Didn't invest more time to figure out why.